### ISSUE TITLE
Tag icons not displaying

### ISSUE THREAD

**Issue by supremebeing7** [calagator/calagator#447]
The ruby/python/pizza etc icons aren't displaying on the feed or on individual event pages.

I don't have time at the moment to debug, but a cursory glance looked like the code to display these was still in the repo so that makes me assume icons weren't removed on purpose.

I'm on Chrome for Android.

**Comment by supremebeing7** [comment]
Update: Confirmed icons are not showing up on Chrome for OSX either (so it's not just a mobile thing). If this is indeed a bug and not a feature that has been deprecated, I can look into this and try to fix this week/weekend.

**Comment by supremebeing7** [comment]
Well, I finally had time to investigate and fix this, and when I looked at the site, the icons are showing up fine. I did nothing, and I see no commits related to this, so I think this is the point I start questioning my sanity and whether they were ever broken to begin with. Except that I confirmed with a colleague who was on a totally different machine.

Anyway, whatever, they appear to be working fine, so closing this issue.
